web/tools/check-build-size.sh

```diff
@@ -120,18 +120,18 @@ if (( $LOCAL_FILE_SIZE <= $REMOTE_FILE_SIZE )); then
   # We are good -- the new build is smaller than the old one
   RESULT_STATE=success
   REPORT_CODE=0
-  RESULT_MESSAGE="✅ Excellent! keymanweb.js is $(printf "%'d" $LOCAL_FILE_SIZE) bytes, $(printf "%'d" $DIFF) bytes ($PERCENT_DIFF%) smaller than $REMOTE_VERSION"
+  RESULT_MESSAGE="✅ Excellent! keymanweb.js is $(printf "%'d" $DIFF) bytes ($PERCENT_DIFF%) smaller than $REMOTE_VERSION, at $(printf "%'d" $LOCAL_FILE_SIZE) bytes"
```

Comment:
Now it's more ambiguous if the size is the new or old size

Reply:
So it is... 🤣🤣🤣

It's not critical though! I will do a one word change and then pretty please can I get an approval? I'd like to merge this administrivial PR!

Reply:
```suggestion
  RESULT_MESSAGE="✅ Excellent! keymanweb.js is $(printf "%'d" $DIFF) bytes ($PERCENT_DIFF%) smaller than $REMOTE_VERSION, now $(printf "%'d" $LOCAL_FILE_SIZE) bytes"
```